CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 0.29.0
+* Internal change to use of FunctionTypeAliasElement (#2051).
```

Comment:
`Internal change to use of ...` => `Internal change to use`?

Reply:
actually it is correct, we were already using it so we were changing the "use of it".  Maybe a reword would be less clumsy, will try to come up with something.

Reply:
ah, gotcha